deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1974,21 +1929,14 @@ extension ChatViewController {
         return preview
     }
 
-    private func reactionsMenu(indexPath: IndexPath) -> UIMenu {
-
+    private func reactionsMenuItems(appendTo: inout [UIMenuElement], indexPath: IndexPath) {
```

Comment:
My gut feeling isn't very happy with using an `inout` and modifying a list of `UIMenuElement` instead of just returning something. It feels like a sideffect: This method builds the reactions menu _and_ oh by the way modifies a parameter. But as always: No hard feelings here :)